pkg/service/service.go

```diff
@@ -21,15 +21,15 @@ type ServiceContainer struct {
 }
 
 var log observ.Logger
-var apiClient utl.APIer
-var out observ.OutputWriter
+var ApiClient utl.APIer
+var Out observ.OutputWriter
```

Comment:
Needed to make these public, so I can mock them in tests
(our lease service relies on global variables)